Package can not be required due to exporting the functions/util directory

index.js exports all files (including directories) under the functions directory. Since the directory functions/util exists it is exports as well.

Requiring this package fails when it attempts to require the exported directory util.

Cannot find module '/ncservice/chreg/node_modules/@nchannel/iqmetrix-dropship-channel/functions/util'

--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
 var fs = require('fs');
 
-var files = fs.readdirSync(__dirname + '/functions');
+var files = fs.readdirSync(__dirname + '/functions').filter(f => f.endsWith('.js'));
 
 for (var i = 0; i < files.length; i++) {
 	var filename = files[i].split('.')[0];

--- a/package.json
+++ b/package.json
@@ -22,5 +22,5 @@
   "scripts": {
     "test": "mocha --timeout 10000 validation/*.spec.js"
   },
-  "version": "0.0.7"
+  "version": "0.0.8"
 }

--- a/package.json
+++ b/package.json
@@ -22,5 +22,5 @@
   "scripts": {
     "test": "mocha --timeout 10000 validation/*.spec.js"
   },
-  "version": "0.0.6"
+  "version": "0.0.7"
 }

